app/services/events.js

```diff
@@ -44,8 +44,27 @@ GameUpApp.factory('events', ['$q', function($q){
     // deferred.resolve;
     // return deferred.promise;
     return gamesList;
+  }
 
+  service.getEvent = function(eventId){
+    var eventObj = Parse.Object.extend("Event");
```

Comment:
You can use [Markdown](https://help.github.com/articles/github-flavored-markdown/) in your comments.

```js
function nicer_looking_code_blocks(){
   // Yay
}
```